CONTRIBUTING.md

```diff
@@ -0,0 +1,66 @@
+# Contributing to Matterpoll
+
+Thank you for your interest in contributing! Join the [**Matterpoll channel**](https://community.mattermost.com/core/channels/matterpoll) on the Mattermost community server for discussion about this plugin.
+
+
+## Reporting Issues
+
+If you think you found a bug, [please use the GitHub issue tracker](https://github.com/matterpoll/matterpoll/issues/new?template=bug_report.md) to open an issue. When opening an issue, please provide the required information in the issue template.
+
+
+## Translating strings
+
+Matterpoll supports localization to various languages. We as maintainers rely on contributors to help with the translations.
+
+The localization process is defined below:
+- During development, new translation strings may be added or existing ones updated.
+- When a new version is planned to release soon, a maintainer opens an issue informing about this. The maintainer will ping all translation maintainer to inform them about this.
+- Translation maintainers submit PRs with new translations, which may get reviewed by other translators.
+- After all translation PRs are merged, the new version is released.
+
+### Translation Maintainers
+
+- France: [@ldidry](https://github.com/ldidry)
+- German: [@hanzei](https://github.com/hanzei)
+- Japanese: [@kaakaa](https://github.com/kaakaa/)
+
+### Translation Process for Existing Languages
+
+1. Ensure all translation messages are correctly extracted:
```

Comment:
It's better to add the process of installing `go-i18n`, or add new section about requirements for translating.

Reply:
Added a paragraph about this.